### ISSUE TITLE
Execute .sln and .csproj

### ISSUE THREAD

**Issue by SamuelvVelzen** [takuya-takeuchi/FaceRecognitionDotNet#89]
Hi Takuya, 

It's me again. After sometime I decided to work again on the project. However this time I coded in Visual Studio. But Visual Studio automatically runs the .sln version and when I did this, it gave the errors:  "Could not copy the file "/Users/Samuel/.nuget/packages/dlibdotnet/19.18.0.20200301/build/../runtimes/osx-AnyCPU/native/libDlibDotNetNative.dylib" and also "Could not copy the file "/Users/Samuel/.nuget/packages/dlibdotnet/19.18.0.20200301/build/../runtimes/osx-AnyCPU/native/libDlibDotNetNativeDnn.dylib".

But when I run the .cspoj in visual studio code the project works. So I am wondering if you know about this error and is it possible to fix. Using OSX Mojave 10.14.6.

**Comment by takuya-takeuchi** [comment]
@SamuelvVelzen 
It is not bug of nuget package.
Could you try build on x64 configuration rather than AnyCPU?

**Comment by SamuelvVelzen** [comment]
I tried it but on Mac it doesn't seem possible to change to a different platform. So I went to my nugget folder and copied the osx-x64 map and renamed it to osx-AnyCpu. When I did this the project seems to work. I think it is because Windows tries to push the default variant. I did ask them how I could fix this issue but they didn't get back to me yet.

But whenever there is a new update I have to redo this process again. So I was wondering if it was possible to include in the nugget package a folder called osx-AnyCpu with the same content as the osx-x64 map?
